packages/dashboard-hardhat-plugin/test/index.test.ts

```diff
@@ -0,0 +1,82 @@
+import { assert, expect } from "chai";
+import { it, describe, before, after } from "mocha";
+import { TASK_COMPILE } from "hardhat/builtin-tasks/task-names";
+import { HardhatError } from "hardhat/internal/core/errors";
+import {
+  IncompatibleHardhatBuildInfoFormatError,
+  IncompatibleHardhatVersionError
+} from "@truffle/from-hardhat";
+import path from "path";
+import { resetHardhatContext } from "hardhat/plugins-testing";
+import { useEnvironment } from "../helpers";
```

Comment:
```suggestion
import { useEnvironment } from "./helpers";
```

Reply:
Good spot @cds-amal, I changed this locally at the last minute (doh). Resolved it in a separate commit so I'll go ahead and close this one without committing.